src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -257,8 +257,10 @@ namespace OutputReportPredefined {
         s->pdstMech = newPreDefSubTable(state, s->pdrEquip, "Central Plant");
 
         s->pdchMechType = newPreDefColumn(state, s->pdstMech, "Type");
-        s->pdchMechNomCap = newPreDefColumn(state, s->pdstMech, "Nominal Capacity [W]");
-        s->pdchMechNomEff = newPreDefColumn(state, s->pdstMech, "Nominal Efficiency [W/W]");
+        s->pdchMechNomCap = newPreDefColumn(state, s->pdstMech, "Reference Capacity [W]");
+        s->pdchMechNomEff = newPreDefColumn(state, s->pdstMech, "Reference Efficiency [W/W]");
```

Comment:
OK. This is heading change. This heading is consistent with input field in the idd.